### ISSUE TITLE
[BUG] Bouton "tester la liaison..." non fonctionnel 

### ISSUE THREAD

**Issue by Emilien52** [CraftMyWebsite/package-minecraft#10]
Bouton pour tester la bonne de liaison de cmwlink avec le site ne fonctionne pas 

![image](https://github.com/user-attachments/assets/2cb0e9b9-b67a-49f1-bcfa-e4eddc3bb129)

**Comment by Emilien52** [comment]
Up ! 

Le problème semble persister sur la alpha-07

**Comment by Zomblard** [comment]
Hello, rien à voir avec la alpha-07 les packages sont indépendants. 
De mon côté RAS aprioris. sur quel bouton tu clique ?

**Comment by Emilien52** [comment]
Sur ce bouton ![Screenshot_20250225_122030_Chrome.jpg](https://github.com/user-attachments/assets/41f50ebc-ceef-4ba1-a673-7e0eae4d8b07)

**Comment by Zomblard** [comment]
Oui, mais j'ai vérifier c'est normal, le package n'a pas été mis à jour depuis longtemps.
J'ai déjà fait les fix il y a quelque mois, on publiera une mise à jour dans la journée

**Comment by Zomblard** [comment]
Tester et fonctionnel sur la version 1.1.0 du package :)
